src/sentry/static/sentry/app/views/settings/organizationRelay/list/cardHeader.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {t, tct} from 'app/locale';
+import Button from 'app/components/button';
+import ButtonBar from 'app/components/buttonBar';
+import QuestionTooltip from 'app/components/questionTooltip';
+import DateTime from 'app/components/dateTime';
+import {IconEdit, IconDelete} from 'app/icons';
+import space from 'app/styles/space';
+import {Relay} from 'app/types';
+import Clipboard from 'app/components/clipboard';
+
+type Props = Relay & {
+  onEditKey: (publicKey: Relay['publicKey']) => () => void;
+  onDeleteKey: (publicKey: Relay['publicKey']) => () => void;
```

Comment:
Could these be `onEdit` and `onDelete` or are there different kinds of things that could be edited and deleted here?